ipatests/test_xmlrpc/test_krbtpolicy.py

```diff
@@ -125,12 +154,93 @@ class test_krbtpolicy(Declarative):
                 value=user1,
                 summary=None,
                 result=dict(
-                    krbmaxticketlife=[u'3600'],
                     krbauthindmaxticketlife_pkinit=[u'3600'],
                 ),
             ),
         ),
 
+
+        dict(
+            desc='Update user ticket policy for auth indicator otp',
+            command=('krbtpolicy_mod', [user1],
+                     dict(krbauthindmaxticketlife_otp=3600)),
+            expected=dict(
+                value=user1,
+                summary=None,
+                result=dict(
+                    krbauthindmaxticketlife_pkinit=[u'3600'],
+                    krbauthindmaxticketlife_otp=[u'3600'],
+                ),
+            ),
+        ),
+
+        dict(
+            desc='Update user ticket policy for auth indicator radius',
+            command=('krbtpolicy_mod', [user1],
+                     dict(krbauthindmaxticketlife_radius=1)),
+            expected=dict(
+                value=user1,
+                summary=None,
+                result=dict(
+                    krbauthindmaxticketlife_otp=[u'3600'],
+                    krbauthindmaxticketlife_pkinit=[u'3600'],
+                    krbauthindmaxticketlife_radius=[u'1'],
+                ),
+            ),
+        ),
+
+        dict(
+            desc='Update user ticket policy for auth indicator hardened',
+            command=('krbtpolicy_mod', [user1],
+                     dict(krbauthindmaxticketlife_hardened=2147483647)),
+            expected=dict(
+                value=user1,
+                summary=None,
+                result=dict(
+                    krbauthindmaxticketlife_otp=[u'3600'],
+                    krbauthindmaxticketlife_pkinit=[u'3600'],
+                    krbauthindmaxticketlife_radius=[u'1'],
+                    krbauthindmaxticketlife_hardened=[u'2147483647'],
+                ),
+            ),
+        ),
+
+        dict(
+            desc='Try updating invalid value auth indicator hardened',
+            command=('krbtpolicy_mod', [user1],
+                     dict(krbauthindmaxticketlife_hardened=u'abc123')
+                     ),
+            expected=errors.ConversionError(
+                name='hardened_maxlife', error='must be an integer'),
+        ),
+
```

Comment:
Each validation test (min val, max val, invalid format) can be done for all the maxticketlife (otp/radius/pkinit/hardened)